### ISSUE TITLE
"current follower url undefine" in competitor_users mode

### ISSUE THREAD

**Issue by gS32tom** [social-manager-tools/instagram-bot.js#76]
### Versions

Replace the values below with your own:

- **Bot Version:** v0.8.2
- **Node Version:** v8.11.1
- **Operating System:** Rasbian 9
- **Browser:** Chromium (running headless)

### Expected Behavior

Please describe the program's expected behavior.

The bot should get the follower list from the competitor, visit a followers page and like multiple photos, get to the next followers page and repeat.

### Actual Behavior

The bot is able to login and seems to be able to visit the competitors profile, but it fails to extract a followers profile url. I previously got an "UnhandledPromiseRejectionWarning: Error: Protocol error (Page.navigate): Invalid parameters url: string value expected" with v0.8.0 but I was able to get rid of it after updating to 0.8.2 and removing previous logs.

```
[INFO] utils: Bot is updated! :D
[INFO] login: loading...
[INFO] login: open_loginpage
[INFO] login: set_username
[INFO] login: set_password
[INFO] login: submit
[INFO] utils: Cheese! Screenshot!
[INFO] login: checkerrors
[INFO] login: password is correct
[INFO] utils: Cheese! Screenshot!
[INFO] login: login_status is 1
[INFO] twofa: instagram request pin (bad location)?
[INFO] twofa: no, try second verify
[INFO] twofa: status location: 0
[INFO] twofa: instagram request pin (2fa enabled)?
[INFO] twofa: no, bot is at work (started)... Wait...
[INFO] twofa: starting current mode
[INFO] utils: Cheese! Screenshot!
[INFO] twofa: status: 0
[INFO] like: competitor_users
[INFO] like: time night: 1123
[INFO] like: loading... Sun Aug 26 2018 11:23:05 GMT+0200 (CEST)
[INFO] like: cache array size 0
[INFO] like: current account removed
[INFO] utils: Cheese! Screenshot!
[INFO] like: get followers
[INFO] like: scroll action
[DEBUG] like: array followers 
[INFO] like: current follower url undefined
[WARNING] like: error follower url.
(node:6128) UnhandledPromiseRejectionWarning: Error: Protocol error (Page.navigate): Invalid parameters url: string value expected
    at Promise (/home/pi/instagram-bot.js/node_modules/puppeteer/lib/Connection.js:200:56)
    at new Promise (<anonymous>)
    at CDPSession.send (/home/pi/instagram-bot.js/node_modules/puppeteer/lib/Connection.js:199:12)
    at navigate (/home/pi/instagram-bot.js/node_modules/puppeteer/lib/Page.js:538:39)
    at Page.goto (/home/pi/instagram-bot.js/node_modules/puppeteer/lib/Page.js:518:7)
    at Likemode_competitor_users.open_follower_account (/home/pi/instagram-bot.js/node_modules/instagrambotlib/modules/mode/likemode_competitor_users.js:105:28)
    at <anonymous>
(node:6128) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6128) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```
### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. run: npm run clean inside the bots folder
2. configure the bot to run in likemode_competitor_users and chose a competitor (in my case that competitor has 28.2k followers)
3. Start the bot with: node bot.js
4. Get the error above.

### config.js (remember to remove password)
```
module.exports = {
    // InstagramBot.js Configs
    "debug": true,
    "login": true,
    "ui": false, // only for social-manager-tools

    // Instagram Account
    "instagram_username": "removed", // without @
    "instagram_password": "removed",
    "instagram_hashtag": ["muraleshunter", "like4like", "follow4follow"], //without #
    "instagram_pin": "sms", //method to receive pin (email or sms)

    // BOT Configs
    // [WORK] likemode_classic          - select random hashtag from config list and like 1 random photo (of last 20) | 400-600 like/day.
    // [WORK] likemode_realistic        - select random hashtag from config list, like fast 10-12 photo and sleep 15-20min. Sleep at night
    // [WORK] likemode_superlike        - select random hashtag from config list and like 3 random photo of same user | 400-600 like/day.
    // [WORK] likemode_competitor_users - select account, select random followers, like 10-12 photo and sleep 15-20min.
    // [WORK] comment_mode              - leave a comment under the photo via hashtag.
    // [TODO] fdfmode_classic           - follow user from random hashtag and defollow after 1h | 300 follow-defollow/day.
    // [TODO] fdfmode_defollowall       - defollow all your following (not defollow users in whitelist) | 30 defollow/hour.
    "bot_mode": "likemode_competitor_users",
    "bot_likeday_min": 800,      // work only in likemode_classic
    "bot_likeday_max": 900,      // work only in likemode_classic. In 2018 limit instagram is descreased to 400-600/day :(
    "bot_sleep_night": true,     // if you need stop bot in night 
    "bot_start_sleep": "7:00",   // sleep from 00:00 to 7:00 am, work in likemode_realistic, comment_mode, likemode_superlike and likemode_competitor_users
    "bot_fastlike_min": "15",    // min minutes of sleep after like 10-12 photo, work in likemode_realistic and likemode_superlike (superlike add 3 like/user, set 50min)
    "bot_fastlike_max": "20",    // max minutes of sleep after like 10-12 photo, work in likemode_realistic and likemode_superlike (superlike add 3 like/user, set 60min)
    "bot_superlike_n": 3,        // superlike number of likes for user
    "bot_followday": 300,        // follow-defollow user/day, work in fdfmode_classic and fdfmode_defollowall
    "bot_userwhitelist": [""],   // usernames without @ | don't defollow this users on follow-defollow mode, work in fdfmode_classic and fdfmode_defollowall
    "comment_mode": {
        "comments": {
            "type": "array",
            "source": [],
        }
    },
    "likemode_competitor_users": {
        "account": "removed", // without @
    },

    // Puppeteer Configs
    "chrome_headless": true,
    "chrome_options": ["--disable-gpu", "--no-sandbox", "--window-size=1920x1080"],
    "executable_path": "/usr/bin/chromium-browser", // example for Mac OS: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome

    // LOG
    "pin_path":"./loginpin.txt",
    "log_path":"./logs/debug.log",
    "logerr_path":"./logs/errors.log",
    "screenshot_path":"./logs/screenshot/",
    "log": {
        "drivers": ["console"], // slack or console
        "screenshot": false,  // disable or enable screenshot in logs folder
        "channels": {
            "console": "",
            "slack": {
                "webhook": ""
            }
        }
    }
};
```

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot. 
![like_account_page](https://user-images.githubusercontent.com/4746531/44626961-2afee780-a926-11e8-892c-d42c716f0454.jpg)
![login_checkerrors](https://user-images.githubusercontent.com/4746531/44626962-2afee780-a926-11e8-9e3e-e77727b9f377.jpg)
![login_submit](https://user-images.githubusercontent.com/4746531/44626963-2b977e00-a926-11e8-8be5-21063052e34a.jpg)
![twofa_check_nopin](https://user-images.githubusercontent.com/4746531/44626964-2b977e00-a926-11e8-9ee9-8e7070d0a5a0.jpg)

**Comment by ptkdev** [comment]
Thanks for report, i fix this in next release.

**Comment by ptkdev** [comment]
Fix available in release **v0.9.0**

<a href="http://patreon.ptkdev.io" alt="Patreon Backer"><img src="https://img.shields.io/badge/donate-patreon-F87668.svg"></a> <a href="http://paypal.ptkdev.io" alt="Paypale Donate"><img src="https://img.shields.io/badge/donate-paypal-46AFE0.svg"></a>  <a href="http://coffee.ptkdev.io" alt="Ko-fi Donate"><img src="https://img.shields.io/badge/buy%20me-coffee-4B788C.svg"></a>
